core/src/main/scala/org/apache/spark/deploy/worker/DriverWrapper.scala

```diff
@@ -38,8 +39,10 @@ object DriverWrapper {
        */
       case workerUrl :: userJar :: mainClass :: extraArgs =>
         val conf = new SparkConf()
-        val rpcEnv = RpcEnv.create("Driver",
-          Utils.localHostName(), 0, conf, new SecurityManager(conf))
+        val host: String = Utils.localHostName()
```

Comment:
I think it is reasonable to use `Utils.localHostName()`. But what is `spark.driver.host` for?